File: tests/runtime/many-arguments/runner.go
package runner

import (
	"context"

	wrpc "wrpc.io/go"

	"driver/runner/test/many_arguments/to_test"
)

func Run(ctx context.Context, c wrpc.Invoker) error {
	return to_test.ManyArguments(ctx, c, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16)
}

File: tests/runtime/rust/resource_into_inner/runner.rs
use crate::runner::test::resource_into_inner::to_test::test;

pub async fn run(
    wrpc: &impl ::wit_bindgen_wrpc::wrpc_transport::Invoke<Context = ()>,
) -> ::wit_bindgen_wrpc::anyhow::Result<()> {
    test(wrpc, ()).await?;
    Ok(())
}
